### ISSUE TITLE
Could not transfer metadata: Read timed out

### ISSUE THREAD

**Issue by danielcompton** [clojars/clojars-web#546]
I'm not really sure what happened here, but I got an error on deploying. However it seems like the deploy was actually successful. Another user has reported a similar error on Slack I think.

```
$ lein release
On branch master
Your branch is up-to-date with 'origin/master'.
nothing to commit, working directory clean
[master 99152db] Version 1.3.1
 1 file changed, 1 insertion(+), 1 deletion(-)
Created /Users/danielcompton/Developer/clojure/clojure-postmark/target/postmark-1.3.1.jar
Wrote /Users/danielcompton/Developer/clojure/clojure-postmark/pom.xml
Need to sign 2 files with GPG
[1/2] Signing /Users/danielcompton/Developer/clojure/clojure-postmark/target/postmark-1.3.1.jar with GPG
[2/2] Signing /Users/danielcompton/Developer/clojure/clojure-postmark/pom.xml with GPG
Sending org/clojars/danielcompton/postmark/1.3.1/postmark-1.3.1.jar (6k)
    to https://clojars.org/repo/
Sending org/clojars/danielcompton/postmark/1.3.1/postmark-1.3.1.pom (3k)
    to https://clojars.org/repo/
Sending org/clojars/danielcompton/postmark/1.3.1/postmark-1.3.1.jar.asc (1k)
    to https://clojars.org/repo/
Sending org/clojars/danielcompton/postmark/1.3.1/postmark-1.3.1.pom.asc (1k)
    to https://clojars.org/repo/
Retrieving org/clojars/danielcompton/postmark/maven-metadata.xml (1k)
    from https://clojars.org/repo/
Sending org/clojars/danielcompton/postmark/maven-metadata.xml (1k)
    to https://clojars.org/repo/
Could not transfer metadata org.clojars.danielcompton:postmark/maven-metadata.xml from/to releases (https://clojars.org/repo/): Read timed out
Failed to deploy metadata: Could not transfer metadata org.clojars.danielcompton:postmark/maven-metadata.xml from/to releases (https://clojars.org/repo/): Read timed out
~/D/c/clojure-postmark (master) $ lein deploy
Created /Users/danielcompton/Developer/clojure/clojure-postmark/target/postmark-1.3.1.jar
Wrote /Users/danielcompton/Developer/clojure/clojure-postmark/pom.xml
Need to sign 2 files with GPG
[1/2] Signing /Users/danielcompton/Developer/clojure/clojure-postmark/target/postmark-1.3.1.jar with GPG
[2/2] Signing /Users/danielcompton/Developer/clojure/clojure-postmark/pom.xml with GPG
Sending org/clojars/danielcompton/postmark/1.3.1/postmark-1.3.1.jar (6k)
    to https://clojars.org/repo/
Sending org/clojars/danielcompton/postmark/1.3.1/postmark-1.3.1.pom (3k)
    to https://clojars.org/repo/
Sending org/clojars/danielcompton/postmark/1.3.1/postmark-1.3.1.jar.asc (1k)
    to https://clojars.org/repo/
Sending org/clojars/danielcompton/postmark/1.3.1/postmark-1.3.1.pom.asc (1k)
    to https://clojars.org/repo/
Retrieving org/clojars/danielcompton/postmark/maven-metadata.xml (1k)
    from https://clojars.org/repo/
Sending org/clojars/danielcompton/postmark/maven-metadata.xml (1k)
    to https://clojars.org/repo/
Could not transfer metadata org.clojars.danielcompton:postmark/maven-metadata.xml from/to releases (https://clojars.org/repo/): Access denied to: https://clojars.org/repo/org/clojars/danielcompton/postmark/maven-metadata.xml, ReasonPhrase: Forbidden - redeploying non-snapshots is not allowed (see http://git.io/vO2Tg).
Failed to deploy metadata: Could not transfer metadata org.clojars.danielcompton:postmark/maven-metadata.xml from/to releases (https://clojars.org/repo/): Access denied to: https://clojars.org/repo/org/clojars/danielcompton/postmark/maven-metadata.xml, ReasonPhrase: Forbidden - redeploying non-snapshots is not allowed (see http://git.io/vO2Tg).
$ 
```

**Comment by tobias** [comment]
I looks like this is being caused by the uploads to cloudfiles taking too long. I suspect the default aether timeout is 10s, and, depending on the size of the jars and the speed of clouldfiles, we're going over that (I was able to recreate the problem with a test app using a 23kb jar).

I've disabled the uploads for now, since we're not yet using cloudfiles to serve the repo. 

The real fix may be to move cloudfiles uploads out to a queue that is processed async. We'll want that queue to be durable, so https://github.com/Factual/durable-queue may be a good fit. We'll also want to move the tmp repos out of `/tmp`, and into a dir that won't be cleared on reboot (with a cronjob to clear old repos).
